Vungle/src/main/java/com/mopub/mobileads/VungleRewardedVideo.java

```diff
@@ -253,27 +270,17 @@ public void onAdEnd(String placementId) {
         @Override
         public void onAdClick(String placementId) {
             if (mPlacementId.equals(placementId)) {
-                if (mInteractionListener != null) {
-                    mInteractionListener.onAdClicked();
-                }
-
+                MoPubRewardedVideoManager.onRewardedVideoClicked(VungleRewardedVideo.class, mPlacementId);
                 MoPubLog.log(getAdNetworkId(), CLICKED, ADAPTER_NAME);
             }
         }
 
         @Override
         public void onAdRewarded(String placementId) {
-            if (mPlacementId.equals(placementId)) {
-                MoPubLog.log(getAdNetworkId(), SHOULD_REWARD, ADAPTER_NAME, MoPubReward.NO_REWARD_AMOUNT,
-                        MoPubReward.NO_REWARD_LABEL);
-                // Vungle does not provide a callback when a user should be rewarded.
-                // You will need to provide your own reward logic if you receive a reward with
-                // "NO_REWARD_LABEL" && "NO_REWARD_AMOUNT"
-                if (mInteractionListener != null) {
-                    mInteractionListener.onAdComplete(MoPubReward.success(MoPubReward.NO_REWARD_LABEL,
-                            MoPubReward.NO_REWARD_AMOUNT));
-                }
-            }
+            MoPubRewardedVideoManager.onRewardedVideoClicked(VungleRewardedVideo.class,
```

Comment:
not sure if this belongs here

Reply:
Pls replace this event implementaion with:
```
MoPubLog.log(getAdNetworkId(), SHOULD_REWARD, ADAPTER_NAME, MoPubReward.NO_REWARD_AMOUNT,
        MoPubReward.NO_REWARD_LABEL);
// Vungle does not provide a callback when a user should be rewarded.
// You will need to provide your own reward logic if you receive a reward with
// "NO_REWARD_LABEL" && "NO_REWARD_AMOUNT"
MoPubRewardedVideoManager.onRewardedVideoCompleted(VungleRewardedVideo.class,
        mPlacementId,
        MoPubReward.success(MoPubReward.NO_REWARD_LABEL,
                MoPubReward.NO_REWARD_AMOUNT));
```